tests/arch/run-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+echo "Running syntax check on arch sim models"
+iverilog -t null ../../techlibs/achronix/speedster22i/cells_sim.v
```

Comment:
Can we wildcard this? It would mean we don't need to remember to update this file when we add new arches

Reply:
Yes, I started this not sure how different arches specified these files, but then it seems like they are fairly consistent so just looking for cells_sim in root or subfolders should work

Reply:
Changed to use find and xargs